src/tests/setup-browser-test-environment.jsx

```diff
@@ -0,0 +1,22 @@
+import '@testing-library/jest-dom/vitest';
```

Comment:
What is the point of this file?

Reply:
This file sets up a browser testing environment for Vitest. Its purpose is to:

- Extend assertions with DOM-specific matchers for easier component testing.
- Automatically clean up rendered components between tests to prevent side effects or leaks.
- Mock Next.js-specific modules (fonts and images) to avoid real dependencies like network loads or optimizations, keeping tests fast and isolated.